test/e2e/workstatus-creation.sh

```diff
@@ -34,15 +34,23 @@ apiVersion: work.open-cluster-management.io/v1
 kind: ManifestWork
 metadata:
   labels:
-    managed-by.kubestellar.io/something: "true"
+    transport.kubestellar.io/something: wds1
```

Comment:
this labels shouldn't be here

Reply:
Same as above. You can see the singletonstatus label on the manifests in lines 47 and 53